rules/windows/command_and_control_new_terms_commonly_abused_rat_execution.toml

```
@@ -19,6 +19,40 @@ index = ["logs-endpoint.events.*", "winlogbeat-*", "logs-windows.*"]
 language = "kuery"
 license = "Elastic License v2"
 name = "First Time Seen Commonly Abused Remote Access Tool Execution"
+note = """
+## Triage and analysis
+
+### Investigating First Time Seen Commonly Abused Remote Access Tool Execution
+
+Remote access software is a class of tools commonly used by IT departments to provide support by connecting securely to users' computers. Remote access is an ever-growing market where new companies constantly offer new ways of quickly accessing remote systems.
+
+At the same pace as IT departments adopt these tools, the attackers also adopt them as part of their workflow to connect into an interactive session, maintain access with legitimate software as a persistence mechanism, drop malicious software, etc.
+
+This rule detects when a remote access tool is seen in the environment for the first time in the last 15 days, enabling analysts to investigate and enforce the correct usage of such tools.
+
+#### Possible investigation steps
+
+- Investigate the process execution chain (parent process tree) for unknown processes. Examine their executable files for prevalence, whether they are located in expected locations, and if they are signed with valid digital signatures.
+- Check if the execution of the remote access tool is approved by the organization's IT department.
+- Investigate other alerts associated with the user/host during the past 48 hours.
+- Contact the account owner and confirm whether they are aware of this activity.
+  - If the tool is not approved for use in the organization, the employee could have been tricked into installing it and providing access to a malicious third party. This third party could be attempting to scam the end-user or gain access to the environment through social engineering.
```

Nit: This isn't exactly a step or action the reader can take; it just describes a possible scenario. Maybe:
```suggestion
  - If the tool is not approved for use in the organization, the employee could have been tricked into installing it and providing access to a malicious third party. Investigate whether this third party could be attempting to scam the end-user or gain access to the environment through social engineering.
```